tests/app/main/views/test_alert.py

```diff
@@ -73,3 +73,34 @@ def test_alert_says_active_alert_is_active(client_get, mocker):
     html = client_get('alerts/21-apr-2021')
     # no "Stopped sending at ..." h2
     assert html.find('main h2') is None
+
+
+@pytest.mark.parametrize('url, expected_href_attribute', (
+    ('gov.uk', 'http://gov.uk'),
+    ('gov.uk/alerts', 'http://gov.uk/alerts'),
+    ('http://gov.uk', 'http://gov.uk'),
+    ('HTTP://GOV.UK', 'HTTP://GOV.UK'),
+    ('https://gov.uk', 'https://gov.uk'),
+    ('https://www.gov.uk', 'https://www.gov.uk'),
+    (
+        'check-for-flooding.service.gov.uk/target-area/055FWFPUNE07',
+        'http://check-for-flooding.service.gov.uk/target-area/055FWFPUNE07',
+    ),
+    (
+        'https://check-for-flooding.service.gov.uk/target-area/055FWFPUNE07',
+        'https://check-for-flooding.service.gov.uk/target-area/055FWFPUNE07',
+    ),
+))
+def test_urls_in_alerts_are_clickable(client_get, mocker, url, expected_href_attribute):
```

Comment:
Could we make do with just one test that the happy path for autolinking works? [It looks like we're testing all the edge cases in utils](https://github.com/alphagov/notifications-utils/pull/971/files#diff-f28b3ed23223b000771c34964f74ea35379b271cc8b4fe4191cd0c637b621fbd), so I'm not sure what value there is in testing a selection of them again.

Do we need them all?

Reply:
Yup, fair point, I’ve cut it down to one simple case and one complex case.